Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -100,37 +100,34 @@ protected void Connect()
             }
         }
 
-        private void DeclareDefaultErrorQueue(IModel model)
+        private static void DeclareAndBindErrorExchangeWithErrorQueue(IModel model, string exchangeName, string queueName, string routingKey)
         {
-            if (!errorQueueDeclared)
-            {
-                model.QueueDeclare(
-                    queue: conventions.ErrorQueueNamingConvention(),
-                    durable: true,
-                    exclusive: false,
-                    autoDelete: false,
-                    arguments: null);
-                errorQueueDeclared = true;
-            }
+            model.QueueDeclare(
+                queue: queueName,
+                durable: true,
+                exclusive: false,
+                autoDelete: false,
+                arguments: null);
+
+            model.ExchangeDeclare(exchangeName, ExchangeType.Topic, durable: true);
```

Comment:
Changing exchange from direct to topic is not backward compatible.

Reply:
Changed back to direct, I realised topic exchange is not needed there, as there will be binding for every routing key from the error quueue to error exchange, so no wildcard characters needed...